### PR TITLE
changes readme.md to enable users

### PR DESCRIPTION
**Pull request by szuecs**
[doc] add public Google Group
[doc] add skipper-plugins
[doc] explain better requirements for kubernetes integration
[doc] fix godoc link to main godoc page
[doc] fix skrop url and skoap hint

### CODE REVIEW

**Comment by szuecs** [comment]
👍

File: readme.md [hunk 1]
```diff
@@ -26,16 +26,19 @@ The Skoap filters can be found currently in the branch called 'skoap-migration'.
 - updates routing rules without downtime, while supporting multiple types of data sources — including
   [etcd](https://github.com/coreos/etcd), [Innkeeper](https://github.com/zalando/innkeeper), static files and
   custom configuration sources
-- can serve as a Kubernetes Ingress implementation in combination with a controller; [see example](https://github.com/zalando-incubator/kube-ingress-aws-controller)
+- can serve as a
+  [Kubernetes Ingress implementation](https://godoc.org/github.com/zalando/skipper/dataclients/kubernetes)
+  in combination with a controller that will route public traffic to
+  your skipper fleet; [see AWS example](https://github.com/zalando-incubator/kube-ingress-aws-controller)
```
Comment (aryszka) [inline readme.md]: will it route the public traffic to the skipper fleet or to the cluster?

**Comment by aryszka** [comment]
:+1:

File: readme.md (hunk 1 above)
Comment (szuecs) [reply readme.md]: The kube-ingress-aws-controller will add ALB with targetGroup skipper-ingress. Maybe this is missleading?
Any idea how to improve?

Comment (aryszka) [reply readme.md]: maybe it's just me. How about:

```
can serve as a
[Kubernetes Ingress implementation](https://godoc.org/github.com/zalando/skipper/dataclients/kubernetes)
in combination with a controller, [see AWS example](https://github.com/zalando-incubator/kube-ingress-aws-controller);
```

...? I think this is enough detail for the readme level.

Comment (szuecs) [reply readme.md]: The problem I see is that people have to understand what our kube-ingress-aws-controller does in order to use skipper in their setup. I hope the added words will explain this in short.

Comment (aryszka) [reply readme.md]: ok, sounds good

**Pull request merged**